changelog.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
```

Comment:
Adjust chnagelog to
```md
# Changelog

## Unreleased
- Initial release
```

Reply:
5ad9051d3e49e58b75cf355556b06722e662d5a1

Reply:
Add
```md
## Unreleased
- Initial release
```

Reply:
4427c81efb647664171d78896ba2417c6ba505b2